templates/common/sdk_delete_post_request.go.tpl

```diff
@@ -2,8 +2,8 @@
 if err == nil {
         if _, err := rm.sdkFind(ctx, r); err != ackerr.NotFound {
                 if err != nil {
-                        return err
```

Comment:
nit: maybe run go fmt on both the template files and lets stick with that. indentation keeps changing between PRs

Reply:
auto formatted :)